projects/Amlogic/bootloader/install

```diff
@@ -0,0 +1,125 @@
+                       --bl301 $FIPDIR/gxb/bl301.bin \
+                       --bl31 $FIPDIR/gxb/bl31.bin \
+                       --bl33 $(get_build_dir $BOOTLOADER)/u-boot.bin $DESTDIR/fip.bin
+
+    $FIPDIR/fip_create --dump $DESTDIR/fip.bin
+
+    cat $FIPDIR/gxb/bl2.package $DESTDIR/fip.bin > $DESTDIR/boot_new.bin
+
+    $FIPDIR/gxb/aml_encrypt_gxb --bootsig --input $DESTDIR/boot_new.bin --output $DESTDIR/u-boot.img
+
+    dd if=$DESTDIR/u-boot.img of=$DESTDIR/u-boot.gxbb bs=512 skip=96
+    dd if=$FUSEDIR/bl1.bin.hardkernel of=$DESTDIR/u-boot.bin.sd.bin conv=fsync,notrunc bs=1 count=442
+    dd if=$FUSEDIR/bl1.bin.hardkernel of=$DESTDIR/u-boot.bin.sd.bin conv=fsync,notrunc bs=512 skip=1 seek=1
+    dd if=$DESTDIR/u-boot.gxbb of=$DESTDIR/u-boot.bin.sd.bin conv=fsync,notrunc bs=512 seek=97
+    ;;
+
+  nanopi-k2)
+    cp $FIPDIR/gxb/bl2.bin $DESTDIR/
+    cp $FIPDIR/gxb/acs.bin $DESTDIR/
+    cp $FIPDIR/gxb/bl21.bin $DESTDIR/
+    cp $FIPDIR/gxb/bl30.bin $DESTDIR/
+    cp $FIPDIR/gxb/bl301.bin $DESTDIR/
+    cp $FIPDIR/gxb/bl31.img $DESTDIR/
+    cp $(get_build_dir $BOOTLOADER)/u-boot.bin $DESTDIR/bl33.bin
+
+    $FIPDIR/blx_fix.sh $DESTDIR/bl30.bin \
+                       $DESTDIR/zero_tmp \
+                       $DESTDIR/bl30_zero.bin \
+                       $DESTDIR/bl301.bin \
+                       $DESTDIR/bl301_zero.bin \
+                       $DESTDIR/bl30_new.bin bl30
+
+    $FIPDIR/fip_create --bl30 $DESTDIR/bl30_new.bin \
+                       --bl31 $DESTDIR/bl31.img \
+                       --bl33 $DESTDIR/bl33.bin \
+                              $DESTDIR/fip.bin
+
+    $FIPDIR/fip_create --dump $DESTDIR/fip.bin
+
+    python $FIPDIR/acs_tool.pyc $DESTDIR/bl2.bin \
+                                $DESTDIR/bl2_acs.bin \
+                                $DESTDIR/acs.bin 0
+
+    $FIPDIR/blx_fix.sh $DESTDIR/bl2_acs.bin \
+                       $DESTDIR/zero_tmp \
+                       $DESTDIR/bl2_zero.bin \
+                       $DESTDIR/bl21.bin \
+                       $DESTDIR/bl21_zero.bin \
+                       $DESTDIR/bl2_new.bin bl2
+
+    cat $DESTDIR/bl2_new.bin $DESTDIR/fip.bin > $DESTDIR/boot_new.bin
+
+    $FIPDIR/gxb/aml_encrypt_gxb --bootsig --input $DESTDIR/boot_new.bin --output $DESTDIR/u-boot.bin.sd.bin
+    ;;
+
+  *)
+    cp $FIPDIR/gxl/bl2.bin $DESTDIR/
+    cp $FIPDIR/gxl/acs.bin $DESTDIR/
+    cp $FIPDIR/gxl/bl21.bin $DESTDIR/
+    cp $FIPDIR/gxl/bl30.bin $DESTDIR/
+    cp $FIPDIR/gxl/bl301.bin $DESTDIR/
+    cp $FIPDIR/gxl/bl31.img $DESTDIR/
+    cp $(get_build_dir $BOOTLOADER)/u-boot.bin $DESTDIR/bl33.bin
+
+    $FIPDIR/blx_fix.sh $DESTDIR/bl30.bin \
+                       $DESTDIR/zero_tmp \
+                       $DESTDIR/bl30_zero.bin \
+                       $DESTDIR/bl301.bin \
+                       $DESTDIR/bl301_zero.bin \
+                       $DESTDIR/bl30_new.bin bl30
+
+    python $FIPDIR/acs_tool.pyc $DESTDIR/bl2.bin $DESTDIR/bl2_acs.bin $DESTDIR/acs.bin 0
```

Comment:
Which Python, 2 or 3? I'd recommend changing this to `${TOOLCHAIN}/bin/python3` and you should be future-proof (as we'll be dropping `Python2` soon-ish), but you should then add `Python3` to `u-boot` `PKG_DEPENDS_TARGET`